getting-started/src/main/kotlin/com/google/cloud/kotlin/demo/EmojifyController.kt

```diff
@@ -83,18 +83,21 @@ fun downloadObject(objectName: String) {
     writeTo.close()
 }
 
+data class EmojifyResponse(val url: String)
+
 @RestController
 class EmojifyController {
     @GetMapping("/emojify")
-    fun emojify(@RequestParam(value = "objectName") objectName: String): String {
+    fun emojify(@RequestParam(value = "objectName") objectName: String): EmojifyResponse? {
 
         var publicUrl: String =
             "https://storage.googleapis.com/${bucket.name}/emojified/emojified-$objectName" // api response
+        val log = Logger.getLogger(EmojifyController::class.java!!.name)
```

Comment:
Can you explain what this is doing?

It looks like it might be more typical to use `Logger.getLogger("javaClass<EmojifyController>())` (this may be what you are doing, I don't know enough kotlin to know or not)

Reply:
I am having a hard time getting the logger to work.

java.util.logging.logger.getLogger() expects the class name  and I think the kotlin version of this code here https://cloud.google.com/appengine/docs/standard/java/logs/ is Logger.getLogger(<class>::class.java.name). The problem is I don't know which of Application.kt and EmojiController I should specify there.